examples/widgets/effectwidget.py

```diff
@@ -51,8 +46,6 @@ def update_effectwidget(self, *args):
                 effects.append(InvertEffect())
             if text == 'mix':
                 effects.append(ChannelMixEffect())
-            if text == 'flash':
-                effects.append(FlashEffect())
```

Comment:
this is not really a PEP8 change, is it?

Reply:
More of flake style issue. `FlashEffect` doesn't seem to exist anywhere, so it complained and I removed it.

Reply:
Ahah true that! I thought it was the `ComparisonWidget` deleted above, but I misread, it's definitely a different class